Cargo.lock

```diff
@@ -127,17 +127,6 @@ dependencies = [
  "slab",
 ]
 
-[[package]]
-name = "async-fs"
-version = "2.1.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "ebcd09b382f40fcd159c2d695175b2ae620ffa5f3bd6f664131efff4e8b9e04a"
-dependencies = [
- "async-lock",
- "blocking",
- "futures-lite",
-]
-
```

Comment:
Why are deps being removed from the lock file as a result of this? :thinking:

Reply:
I executed `cargo machete` (I get use to that tbh), and I found some unused dependencies. I ran test just to be sure that everything is working, but let me know if I miss something  :ok_hand:

Reply:
Actually never mind. Just rebase on latest main after [this](https://github.com/dbus2/zbus/pull/1286) is merged. :)

Reply:
Sorry @zeenix . I've open a clean new one, thanks.

Reply:
I never understood why people create new PRs. It's just more noise. Just gotta force push to the same branch on the remote.